cmd/node/app_test.go

```diff
@@ -423,7 +452,9 @@ func (suite *AppTestSuite) validateBlocksAndATXs(untilLayer types.LayerID) {
 
 		for i := types.LayerID(5); i <= untilLayer; i++ {
 			lyr, err := ap.mesh.GetLayer(i)
-			assert.NoError(suite.T(), err, "couldn't get a validated layer from db layer %v", i)
+			if err != nil {
+				log.With().Panic("couldn't get a validated layer from db", i, log.Err(err))
+			}
```

Comment:
wouldn't a `require.NoError()` instead of `assert.NoError()` do the trick here? I'm not sure how it behaves when called in a goroutine, so it's a genuine question.

Reply:
It doesn't work inside a goroutine. But this isn't inside a goroutine. I may have missed that, or I may have just wanted failure here to short-circuit execution of the rest of the test. I fixed it.